docs/JOBS.md

```diff
@@ -1,11 +1,11 @@
-### External Jobs
+### Jobs (Checks that run just once)
```

Comment:
Instead of just Jobs, can we rename to Kuberhealthy Jobs? or khjobs? I don't want users to get confused with kubernetes jobs

Reply:
done